akka-remote/src/main/scala/akka/remote/artery/compress/TopHeavyHitters.scala

```diff
@@ -38,10 +38,9 @@ private[remote] final class TopHeavyHitters[T >: Null](val max: Int)(implicit cl
   /** Returns copy(!) of items which are currently considered to be heavy hitters. */
   def snapshot: Array[T] = {
     val snap = Array.ofDim(max).asInstanceOf[Array[T]]
-    var i: Int = 0
-    val heapIterator = heap.reverseIterator
-    while (heapIterator.hasNext) {
-      val index = heapIterator.next()
+    var i = 0
+    while (i < max) {
+      val index = heap(i)
```

Comment:
We should probably do it in the same order as before, there might be a reason.

@bambuchaAdm can you enlighten us of if the iteration order matters here?